pkg/controller/restore_controller.go

```diff
@@ -402,10 +441,19 @@ func downloadToTempFile(backupName string, backupService cloudprovider.BackupSer
 	if err != nil {
 		return nil, err
 	}
-	glog.V(4).Infof("copied %d bytes", n)
+
+	logContext := logger.WithField("backup", backupName)
+
+	logContext.WithFields(logrus.Fields{
+		"fileName": file.Name(),
+		"bytes":    n,
+	}).Debug("Copied Backup to file")
 
 	if _, err := file.Seek(0, 0); err != nil {
-		glog.V(4).Infof("error seeking: %v", err)
+		logContext.WithFields(logrus.Fields{
```

Comment:
Please don't log an error at Info level, please don't log and return an error, just return it.